src/java2yaml/Steps/RestoreDependency.cs

```diff
@@ -0,0 +1,72 @@
+﻿namespace Microsoft.Content.Build.Java2Yaml
+{
+    using System;
+    using System.IO;
+    using System.Threading.Tasks;
+    using System.Linq;
+
+    public class RestoreDependency : IStep
+    {
+        public RestoreDependency(string path)
+        {
+            RepositoryPath = path;
+        }
+        public string StepName
+        {
+            get { return "RestoreDependency"; }
+        }
+
+        public string RepositoryPath { get; }
+
+
+        public Task RunAsync(ConfigModel config)
+        {
+            return Task.Run(() =>
+            {
+                Guard.ArgumentNotNullOrEmpty(RepositoryPath, nameof(RepositoryPath));
+
+                var projectFiles = FileUtility.GetFilesByName(RepositoryPath, Constants.BuildTool.Maven);

```

Comment:
Do not reuse `IEnumerable`. Or you can reuse it after `ToList()`.

Reply:
added `ToList()`